### ISSUE TITLE
Playing medic tutorial in Russian throws an error and freezes game

### ISSUE THREAD

**Issue by Rokvach** [Regalis11/Barotrauma#9362]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Spotted this issue mentioned in couple places and decided to give it a try

### Reproduction steps

1. Change language to Russian
2. Load up medic tutorial
3. In the first objective you should receive an error and you can't advance

### Bug prevalence

Happens every time I play

### Version

Other

### -

0.18.11.0

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
Coroutine Barotrauma.Tutorials.DoctorTutorial+<UpdateState>d__25 threw an exception: Index (zero based) must be greater than or equal to zero and less than the size of the argument list.
   at System.Text.StringBuilder.AppendFormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.Format(String format, Object[] args)
   at Barotrauma.FormattedLString.RetrieveValue() in <DEV>\Barotrauma\BarotraumaShared\SharedSource\Text\LocalizedString\FormattedLString.cs:line 21
   at Barotrauma.LocalizedString.get_Value() in <DEV>\Barotrauma\BarotraumaShared\SharedSource\Text\LocalizedString\LocalizedString.cs:line 25
   at Barotrauma.InputTypeLString.RetrieveValue() in <DEV>\Barotrauma\BarotraumaShared\SharedSource\Text\LocalizedString\InputTypeLString.cs:line 20
   at Barotrauma.LocalizedString.get_Value() in <DEV>\Barotrauma\BarotraumaShared\SharedSource\Text\LocalizedString\LocalizedString.cs:line 25
   at Barotrauma.WrappedLString.RetrieveValue() in <DEV>\Barotrauma\BarotraumaClient\ClientSource\Text\LocalizedString\WrappedLString.cs:line 24
   at Barotrauma.LocalizedString.get_Value() in <DEV>\Barotrauma\BarotraumaShared\SharedSource\Text\LocalizedString\LocalizedString.cs:line 25
   at Barotrauma.GUIFont.MeasureString(LocalizedString str, Boolean removeExtraSpacing) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\GUI\GUIPrefab.cs:line 233
   at Barotrauma.Tutorials.Tutorial.CreateInfoFrame(LocalizedString title, LocalizedString text, Int32 width, Int32 height, Anchor anchor, Boolean hasButton, Action callback, Action showVideo) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\GameSession\GameModes\Tutorials\Tutorial.cs:line 441
   at Barotrauma.Tutorials.Tutorial.TriggerTutorialSegment(Index index, Object[] args) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\GameSession\GameModes\Tutorials\Tutorial.cs:line 247
   at Barotrauma.Tutorials.DoctorTutorial.UpdateState()+MoveNext() in <DEV>\Barotrauma\BarotraumaClient\ClientSource\GameSession\GameModes\Tutorials\DoctorTutorial.cs:line 258
   at Barotrauma.CoroutineManager.PerformCoroutineStep(CoroutineHandle handle) in <DEV>\Barotrauma\BarotraumaShared\SharedSource\CoroutineManager.cs:line 197
   at Barotrauma.CoroutineManager.IsDone(CoroutineHandle handle) in <DEV>\Barotrauma\BarotraumaShared\SharedSource\CoroutineManager.cs:line 255
```

**Comment by andremgbr** [comment]
In Portuguese Brazilian is happening as well! Same error.

**Comment by pfjhyyj** [comment]
Exactly the same bug, but happens on Chinese (both traditional and simplifed). When changed the language to Japanese things go right.
![$)`EBD%W0$ZBUVF)2LUEH`0](https://user-images.githubusercontent.com/15180163/174212032-870fe2b0-dea4-4a77-80d6-e287328c7667.png)

**Comment by Rokvach** [comment]
Tested, I wasn't able to reproduce the issue anymore after the fix, if nothing is found during review this ticket can be closed

**Comment by Vaselevs** [comment]
I got the same error myself.

Reproduction steps:

1.Switched the language in the menu from Russian to English
2. I went to the doctor's tutorial
3. After the appearance of the first dialog box and a tooltip with the OK button, I switched the language to Russian
4. I clicked on the window with a hint on my next steps (located above the chat in the lower left corner)
5. Got a game crash and an error.

![image](https://user-images.githubusercontent.com/13721274/174286050-8c7ebdee-5b5b-4798-9251-6ce38562afda.png)
![image](https://user-images.githubusercontent.com/13721274/174286093-e138a591-4700-437c-bf35-9435315a229d.png)
[crashreport.txt](https://github.com/Regalis11/Barotrauma/files/8926902/crashreport.txt)
